docs/gke/configs/cluster-kubeflow.yaml

```diff
@@ -16,6 +16,14 @@ imports:
 - path: cluster.jinja
 
 resources:
+  # Deployment manager doesn't support depends on references in template type.
+  # So the two possible work arounds are
+  # 1. Use a single template (.jinja file for all resources) or
+  # 2. Create two separate deployments and launch the boot strapper
+  # after the cluster is created.
+  # 
+  # Two separate deployments doesn't make much sense; we could just use 
```

Comment:
Having `initialNodeCount` and `cpu-pool-initialNodeCount` can be confusing. Can `initialNodeCount` property be removed from here and set to 1 by default in cluster.jinja?

Reply:
Done